tests/secrets/mkdir.test.ts

```diff
@@ -94,6 +95,106 @@ describe('commandMkdir', () => {
       expect(stat2.isDirectory()).toBeTruthy();
     });
   });
+  test('should fail without recursive set', async () => {
+    const vaultName = 'vault' as VaultName;
+    const dirName1 = 'dir1';
+    const dirName2 = 'dir2';
+    const dirNameNested = `${dirName1}/${dirName2}`;
```

Comment:
You should be using `path.join()` when constructing paths like this rather than use a template string. Keep in mind that the fs assumes unix style paths but `path.join()` will switch based on platform. There is a utility that handles `path.join()` for the efs. You'll probably need to check the vaults code to find it.

As a general rule never construct path strings manually like this.